source/_integrations/reolink.markdown

```diff
@@ -0,0 +1,111 @@
+---
+title: Reolink NVR/camera
+description: Instructions on how to integrate Reolink devices (NVR/cameras) into Home Assistant.
+ha_category:
+  - Camera
+ha_iot_class: Local Polling
+ha_release: 2022.12
+ha_domain: reolink
+ha_codeowners:
+  - '@starkillerOG'
+ha_config_flow: true
+ha_platforms:
+  - Camera
+ha_integration_type: integration
+---
+
+The integration allows you to control [Reolink](https://reolink.com/) NVRs or cameras.
+
+{% include integrations/config_flow.md %}
+
+## Known reolink firmware bugs
+
+<div class='note warning'>
+There is a BUG in Reolink NVR firmware: it only sends ONVIF event-notifications if motion happened on the camera connected to its very first (index "0") channel.
+Reolink is aware of this and are working on fixing the problem.
+</div>
+
+<div class='note warning'>
+Reolink doorbell presses only generate ONVIF event-notifications when the doorbell is directly connected to your network.
+When connecting the Reolink doorbell to a NVR, no doorbell press events are generated.
+Reolink is aware of this and a solution is already implemented in beta firmware.
+</div>
```

Comment:
These type of notices tend to go stable in docs

Is there an Reolink page you can link to instead?

Reply:
No there is no page to link to that I am aware of, I got this information from the reolink dev team directly through private messages.
Since I am working on resolving those two bugs together with reolink, I can update the docs when it is fixed.
But I am also willing to just remove the two warnings...

Reply:
Lets present these as limitations for now.  When Reolink updates their firmware they can be updated

Reply:
Alright, I have commited your suggestions to call it limitations.